### DISCUSSION TITLE
Export and Load

### DISCUSSION THREAD

**Discussion by Ozaier** [OndrejNepozitek/Edgar-Unity#116] [Q&A] [answered]
Hi there,

I'm trying to load my dungeon which is created during play mode, using this method:
`ExportMapDescription();`
`LevelDescriptionGrid2D<RoomBase>.LoadFromJson("exportedMapDescription.json");`
but it didn't work,
error like this
`JsonSerializationException: Could not create an instance of type Edgar.Unity.RoomBase. `
i wonder how to load the ".json" file

Thanks!

**Comment by OndrejNepozitek** [comment, marked as answer]
Hello!
I'm sorry but the button was there only for debugging purposes because it allowed me to recreate a level in another environment where I could debug more easily.

Usually, if you want to recreate a level, you remember the seed that you used and is it again, which will lead to the same level being generated.

Or do you have a different goal in mind?

**Reply by Ozaier** [reply]
I got it, so i just need to save the seed then i can recreate a same level. The other data i can save in a different way which can run after the level being recreated.Thanks!
